packages/react-server-cli/gulpfile.babel.js

```diff
@@ -1,8 +1,21 @@
 import eslint from "gulp-eslint";
 import gulp from "gulp";
 import babel from "gulp-babel";
+import jasmine from "gulp-jasmine";
 import logging from "react-server-gulp-module-tagger";
 
+function getSpecGlob (prefix) {
+	// add a wildcard onto the end if no file extension or wildcard
+	// currently present
+	let specGlob = "*[Ss]pec.js";
+	if (!specGlob.endsWith(".js") && !specGlob.endsWith("*")) {
```

Comment:
Oh, just realized this was left outstanding.

@drewpc @doug-wade Should we file a followup ticket for this?